apps/emqx_retainer/etc/emqx_retainer.conf

```diff
@@ -45,32 +45,32 @@ retainer {
   ##    repeat this, until all retianed messages are delivered
   ##
   flow_control {
-    ## The max messages number per read from storage. 0 means no limit
+    ## The messages number per read from storage. 0 means no limit
     ##
     ## Default: 0
-    max_read_number = 0
+    batch_read_number = 0
 
-    ## The max number of retained message can be delivered in emqx per quota_release_interval.0 means no limit
+    ## The number of retained message can be delivered per batch
     ##
     ## Default: 0
-    msg_deliver_quota = 0
+    batch_deliver_number = 0
 
-    ## deliver quota reset interval
+    ## deliver limiter bucket
     ##
     ## Default: 0s
-    quota_release_interval = 0s
+    limiter_bucket_name = retainer
```

Comment:
I don't quite know if it would be easier to understand this configuration here. Why not just configure it here. i.e:
```
dispatch_limiter {
  aggregated.rate = infinity
  aggregated.capacity = infinity
  per_client.rate = infinity
  per_client.capacity = infinity
}
```